core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java

```diff
@@ -237,26 +235,24 @@ public int getViewId() {
 
    @Override
    public void waitForView(int viewId) throws InterruptedException {
-      waitForView(viewId, 0, TimeUnit.SECONDS);
+      waitForView(viewId, Long.MAX_VALUE, TimeUnit.NANOSECONDS);
```

Comment:
It's the same as the `waitForView(0, SECONDS)` we had before, and I'd argue it's easier to read because we don't have to guess that `0` means wait forever.